### PR TITLE
Added boundaries

### LINKED ISSUE #15: UX: Filter on Surrey FSA Boundaries
- Map visual enhancement
- Selection and zoom feature into certain FSA code areas

### PR DESCRIPTION
**Pull request by MartinYTSo**

### CODE REVIEW

**Comment by MartinYTSo** [comment]
Added boundary selection for Surrey
image

When selected a specific neighbourhood, the camera will zoom into the neighbourhood

image

File: app.R [hunk 1]
```diff
@@ -17,6 +17,32 @@ MULTI_SELECT_TOGGLE = TRUE
 # Load data
 cams <- read.csv("data/surrey_desc.csv")
 cams_data <- read.csv("data/surrey_data.csv")
+surrey_fsa<-readLines("data/surrey_boundary_with_FSA.geojson") %>% paste(collapse = "\n")
+
+neighbourhood <- st_read("data/surrey_city_boundary.json", quiet = TRUE) %>%
```
Comment (gouruiyu) [inline app.R]: Is there a reason this part is not reusing `readBoundaries.R`?

File: app.R [hunk 2]
```diff
@@ -100,6 +126,13 @@ ui <- dashboardPage(
       menuItem("Cam Map", tabName = "basemap", icon = icon("camera")),
       menuItem("User Inputs", tabName = "userInputs", icon = icon("user")),
       hidden(
+        selectInput(
+          "neighbourhood_names",
+          label = "Select a Neighbourhood:",
+          choices=(neighbourhood_names),
+          selected= "SURREY"
+
```
Comment (gouruiyu) [inline app.R]: Let's remove the unnecessary line inside function parameters.
Also try to keep a consistent spacing before and after parameter "=".

File: readBoundaries.R [hunk 3]
```diff
@@ -0,0 +1,27 @@
+library(sf)
+library(ggplot2)
+library(dplyr)
+library(lubridate)
+library(tidyverse)
```
Comment (gouruiyu) [inline readBoundaries.R]: Again, if you intend to source this file in `app.R`, try not to include libraries unnecessary to this file.
For example `readBoundaries.R` does not use ggplot2 or lubridate

**Review by gouruiyu — CHANGES_REQUESTED** [review]
Feature lgtm
Left some reviews on the code

**Review by gouruiyu — DISMISSED** [review]
Still some unnecessary empty lines. I will approve it for now but please try to keep them in mind in the future : D
image

**Review by nancywan1004 — DISMISSED** [review]
Just tried the latest version and it works fine now.

**Review by gouruiyu — DISMISSED** [review]
One suggestion for refinement: Make the boundary lines semi-transparent so one can see the road names the are sometimes covered now.
image
Otherwise LGTM! Good job :)

**Comment by MartinYTSo** [comment]
image

Made river semi transparent

**Review by gouruiyu — APPROVED** [review]
LGTM!

**Review by nancywan1004 — APPROVED** [review]

**Pull request merged**